src/config.c

```diff
@@ -522,7 +523,7 @@ int prepare_prefixes(statsite_config *config)
         assert(res != -1);
         config->prefixes_final[i] = final_prefix;
     }
-    return 0;
+    return final_prefix;
```

Comment:
We only capture one final prefix? We should probably enumerate the config on shutdown instead.